tests/test_datasets.py

```diff
@@ -1,23 +1,50 @@
 """Unit tests for loading Graph dataset."""
 
+import geomstats.backend as gs
+import geomstats.datasets.utils as data_utils
 import geomstats.tests
 from geomstats.datasets.graph_data_preparation import Graph
+from geomstats.geometry.hypersphere import Hypersphere
+from geomstats.geometry.special_euclidean import SpecialEuclidean
+from geomstats.geometry.special_orthogonal import SpecialOrthogonal
 
 
-class TestLoadGraph(geomstats.tests.TestCase):
+class TestDatasets(geomstats.tests.TestCase):
     """Test for loading graph-structured data."""
 
-    @geomstats.tests.np_only
     def setUp(self):
-        """Declare the graph by default and the Karate club graph."""
+        """Set up tests."""
         self.g1 = Graph()
         self.g2 = Graph(
-            graph_matrix_path='data'
-                              '/graph_karate/karate.txt',
-            labels_path='data'
-                        '/graph_karate/karate_labels.txt')
+            graph_matrix_path=data_utils.KARATE_PATH,
+            labels_path=data_utils.KARATE_LABELS_PATH)
 
-    @geomstats.tests.np_only
+    def test_load_cities(self):
+        """Test that the cities coordinates belong to the sphere."""
+        sphere = Hypersphere(dim=2)
+        data, _ = data_utils.load_cities()
+        data = sphere.spherical_to_extrinsic(data)
```

Comment:
Really nice that we can now do this! Did you make sure that our `spherical_to_extrinsic` uses the right reference point for the spherical coordinates ? ([0, 0, 1]  should be the north pole but we used [1, 0, 0] I think). Also I think there is a further conversion from latitude to colatitude to do (x -> pi / 2 - x). This is probably not important here, but in a tutorial we should be faithful to geography :)

Reply:
🙏 Thanks! I was battling with this on the tutorial indeed (see PR #782 ).

Reply:
I'll address this in the next PR.